### PR TITLE
Added case study section & remove link to The Hive.

### PR DESCRIPTION
**Pull request by JacOng17**
image
As requested. Added case study section & remove link to The Hive.

### CODE REVIEW

File: pages/feedback-loop.md [hunk 1]
```diff
@@ -47,10 +46,20 @@ in_nav: true
...
+
+  <p>The 15 rangatahi (known as the Hivers) were invited to participate in a co-design hui to identify and create solutions for "How might we enable a two-way communication platform between young people and policy makers?" They were presented with the issue, and went on to design The Hive which included youth centric website, with accessible and relatable content, and a social media drive.</p>
+  
+  <p>Following a successful consultation, using the Biodiversity, The Hivers were consulted on how the results should be released. The methods used, social media, video, info graphics, reflects the consultation methods and the community we wanted to engage with.</p>
+  
+  <p>Through the process The Hivers designed, created and were consulted throughout. Resulting in the entire process been relevant and accessible for the people we are wanted to engage.
```
Comment (mermop) [inline pages/feedback-loop.md]:
```suggestion
  <p>Through the process The Hivers designed, created and were consulted throughout. As a result, the entire process was relevant and accessible for the people we wanted to engage.
```

File: pages/feedback-loop.md [hunk 2]
```diff
@@ -47,10 +46,20 @@ in_nav: true
...
+
+  <a name="Case-Study"></a>
+  <h3>Case Study</h3>
+
+  <p>We have tried to explore and follow this process with The Hive. The Hive is a Ministry of Youth Development (MYD) Youth Voice Project that uses an innovative approach to increasing young people’s participation in the government policy development process. MYD worked in partnership with design agency Curative and 15 rangatahi on this project.</p>
+
+  <p>The 15 rangatahi (known as the Hivers) were invited to participate in a co-design hui to identify and create solutions for "How might we enable a two-way communication platform between young people and policy makers?" They were presented with the issue, and went on to design The Hive which included youth centric website, with accessible and relatable content, and a social media drive.</p>
+  
+  <p>Following a successful consultation, using the Biodiversity, The Hivers were consulted on how the results should be released. The methods used, social media, video, info graphics, reflects the consultation methods and the community we wanted to engage with.</p>
```
Comment (mermop) [inline pages/feedback-loop.md]:
```suggestion
  <p>Following a successful consultation, on the Biodiversity Strategy, The Hivers were consulted on how the results should be released. The methods used - social media, video, and infographics - reflects the consultation methods and the community we wanted to engage with.</p>
```

File: pages/feedback-loop.md [hunk 3]
```diff
@@ -47,10 +46,20 @@ in_nav: true
...
+
+  <a name="Case-Study"></a>
+  <h3>Case Study</h3>
+
+  <p>We have tried to explore and follow this process with The Hive. The Hive is a Ministry of Youth Development (MYD) Youth Voice Project that uses an innovative approach to increasing young people’s participation in the government policy development process. MYD worked in partnership with design agency Curative and 15 rangatahi on this project.</p>
+
+  <p>The 15 rangatahi (known as the Hivers) were invited to participate in a co-design hui to identify and create solutions for "How might we enable a two-way communication platform between young people and policy makers?" They were presented with the issue, and went on to design The Hive which included youth centric website, with accessible and relatable content, and a social media drive.</p>
```
Comment (mermop) [inline pages/feedback-loop.md]:
```suggestion
  <p>The 15 rangatahi (known as the Hivers) were invited to participate in a co-design hui to identify and create solutions for "How might we enable a two-way communication platform between young people and policy makers?" They were presented with the issue, and went on to design The Hive - a youth centric website with accessible and relatable content - and a social media drive.</p>
```

**Review by rosspatel01 — APPROVED** [review]
This looks perfect. Thanks brother

**Pull request merged**